internal/api/auth/token_verifier_test.go

```diff
@@ -68,14 +84,20 @@ func TestOIDCTokenVerifier(t *testing.T) {
 		})
 		assert.NilError(t, err)
 
-		verified, err := v.Verify(ctx, token)
+		verified, claims, err := v.Verify(ctx, token)
 		assert.NilError(t, err)
 		assert.Assert(t, verified == false)
+		assert.DeepEqual(t, claims, (*auth.Claims)(nil))
```

Comment:
I saw the other conversation with David on this check. This is fine to me but if you're still looking for other alternatives I think this could be one using `gotest.tools/v3/assert/cmp`:

```suggestion
		assert.Assert(t, cmp.IsNil(claims))
```

It would return:

    assertion failed: {foo@bar.com false []} (type *auth.Claims) is not nil